<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchItemDialog</class>
 <widget class="QDialog" name="SearchItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidgetItems">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>771</width>
     <height>421</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Item name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Area</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>LibID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Lib name</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>550</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset resource="PropertySkipMark.qrc">
     <normaloff>:/Qtres/OK.png</normaloff>:/Qtres/OK.png</iconset>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>550</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="PropertySkipMark.qrc">
     <normaloff>:/Qtres/Cancel.png</normaloff>:/Qtres/Cancel.png</iconset>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLineEdit" name="lineEditTopName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>By name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>By ID</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxTopID">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSortByID">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort and top</string>
    </property>
    <property name="icon">
     <iconset resource="PropertySkipMark.qrc">
      <normaloff>:/Qtres/SortNumber.png</normaloff>:/Qtres/SortNumber.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSortByName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort and top</string>
    </property>
    <property name="icon">
     <iconset resource="PropertySkipMark.qrc">
      <normaloff>:/Qtres/SortNumber.png</normaloff>:/Qtres/SortNumber.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSortByLibID">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort by LibID</string>
    </property>
    <property name="icon">
     <iconset resource="PropertySkipMark.qrc">
      <normaloff>:/Qtres/SortNumber.png</normaloff>:/Qtres/SortNumber.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="PropertySkipMark.qrc"/>
 </resources>
 <connections/>
</ui>
